src/components/external/DisputeManagement/components/AcceptDisputeFlow/AcceptDisputeFlow.tsx

```diff
@@ -37,8 +37,9 @@ export const AcceptDisputeFlow = ({
         options: {
             onSuccess: useCallback(() => {
                 onAcceptDispute?.();
+                clearStates();
                 setDisputeAccepted(true);
```

Comment:
What happens if `onAcceptDispute` throws an error here? Then the rest of the `onSuccess` logic doesn't run, and I don't think that is what we want in this case. I recommend doing this instead:

```suggestion
                clearStates();
                setDisputeAccepted(true);
                // always attempt to call this callback as the last step
                onAcceptDispute?.();
```